tests/api/test_forwarding.py

```diff
@@ -167,7 +167,9 @@ async def post_success(*args, **kwargs):  # pylint: disable=unused-argument
     ] == [message for source, _, message in caplog.record_tuples if source != "asyncio"]
 
 
-@hypothesis_settings(suppress_health_check=(HealthCheck.function_scoped_fixture,))
+@hypothesis_settings(
+    deadline=None, suppress_health_check=(HealthCheck.function_scoped_fixture,)
+)
```

Comment:
Maybe we can restore the default deadline as we've upgraded CI execution class?

Reply:
Good catch!

Reply:
We're still exceeding the deadline even on a large resource on this test, I leave `deadline=None` as such